packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/index.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  FontManager,
+  Font,
+  FONT_FAMILY_DEFAULT,
+  OPTIONS_DEFAULTS,
+} from '@samuelmeuli/font-manager';
+import React, { useState, useEffect } from 'react';
+import OutsideClickHandler from 'react-outside-click-handler';
+import { FontFamily, SearchFonts, FontLI, List } from './elements';
+import FontList from './List';
+
+export const FontPicker = ({
+  activeFontFamily = FONT_FAMILY_DEFAULT,
+  onChange,
+  apiKey,
+}) => {
+  const [expanded, setExpanded] = useState(false);
+  const [loadingStatus, setLoadingStatus] = useState('loading');
+  const [fontManager, setFontManager] = useState();
+
+  useEffect(() => {
+    const manager = new FontManager(
```

Comment:
i'd init this manager out with noops and then let the effects below set the initial `onChange` and `activeFontFamily`

Reply:
ignore what I said, the way the component is set now makes this bad advice!